frontend/src/lib/arrowProto.ts

```diff
@@ -152,83 +157,153 @@ function getColumns(schema: Schema): string[][] {
   )
 }
 
-function getData(table: Table<any>, columns: string[][]): any[][] {
+function getColumnType(schema: Schema): any {
```

Comment:
Is it possible to bound this return type at all? If the type is super-hairy, could it be described in a function docstring?

Reply:
Done